Initial PR (#1)
greenelab/nature_news_manuscript: content/05.discussion.md

Except Celtic/English and East Asian, all name origins roughly match the expected academic background rate estimated by Nature last authorship.
We find a clear over-representation of Celtic/English names and under-representation of East Asian names.
We also found this same pattern in our Springer data set.
 
After observing name origin biases, we sought to identify a bias in the frequency or content of coverage across countries.
We first looked at possible citation biases for authors with specific country affiliations, and found that most manuscripts cited by Nature news have at least one author affiliated with the United States, United Kingdom, and Germany.
In contrast to the name origins analysis shown previously, we identified that the citation rate of China was not significantly under-enriched by citation.
Interestingly, we find the number of citations affiliated with China is increasing at the same rate compared to Springer and Nature authorships.
The increased citation rate differs from all other countries within the top ten most cited. 

> In constrast, there is no considerable change in Nature authorship but an increase in the articles cited by Nature news.

Author: yes, you are absolutely correct. I don't know what I was getting at here... updated in https://github.com/greenelab/nature_news_manuscript/pull/1/commits/a4b8d0815e823917665b81a43adee15f37a34647

We speculate that news coverage of research articles may be biased towards more extensive and more international pieces of work.

We then focused on identifying whether the news content about a country focused on the scientific output from that country or the country itself as the scientific subject.
We postulated that a difference in citation and mention rates could indicate the difference in a news article’s subject matter.
To achieve this, we identified two sets of countries with a large and consistent difference in their citation and mention rates.
The top "Citation" countries were Germany, Netherlands, and Japan.
The top "Mention" countries were India, Colombia, and the Philippines.
We then found that these two sets of countries were discussed differently.

> The resultant words for "Mention" countries were most related to agriculture; this indicates that the country was likely the article’s subject.

Reviewer: ```suggestion
The resultant words for "Mention" countries were most related to agriculture; suggesting that the country was likely the article’s subject.
```

In contrast, the representative words for "Citation" countries were more diverse in topic, relating to biological, medical, and physics terms.
The difference in discriminative terms between the two country sets is evidence that the news content may focus more on research of a country as a subject than science that comes out of it.

 
Through our comprehensive analysis, we were able to identify how news coverage varies by country, name origin, and gender, and compare it to scientific publishing background rates.

> While we were able to identify a significant gender bias, we found that the rate of female representation in scientific news is increasing and outpacing the scientific publishing field.

Reviewer: ```suggestion
While we found a significant gender bias, the rate of female representation in scientific news is increasing and outpacing authorships on scientific manuscripts.
```

> Furthermore, we identified a significant bias against coverage from those with an East Asian name origin.

Reviewer: ```suggestion
Furthermore, we identified a significant depletion of quotes from scientists with an East Asian name origin and a significant but smaller depletion of authors with an East Asian name origin among the citations of news content.
```
DidI get this right?

## Discussion
 

> Scientific news coverage, unlike academic articles, is accountable to reflecting not only the current state of research, but also the diversity of its readership.

Reviewer: ```suggestion
Scientific news coverage, unlike academic articles, should represent not only the current state of research-related disciplines but also the diversity of its readership.
```

News coverage is the conduit between the academic and public spheres, and consequently shapes the public's view of science and scientists.

> At a minimum, news coverage should match the current academic rates of regional and gender biases.

Reviewer: ```suggestion
Though it would be best for news coverage to promote equitable representation, at a minimum news coverage should match the current rates of regional and gender participation in academia.
```

We then compared this to the authorship statistics from Nature's research articles and a subset of Springer's English language articles.

We first looked at possible gender biases by examining the proportion of all identified quotes that were from a male speaker.
As a comparator, we analyzed the proportion of male authors, which similarly is a measure of scientific participation.
Using computational methods, we performed quote association and gender prediction.
We observe a strong bias towards male participation across both authorship and quotes.
We also identify a gender bias between first and last authors, previously shown in [CITE].
Interestingly, we found that while the rate of male quotes and male authorship are decreasing over time, quotation rate is decreasing faster than authorship rates.
We then tried to identify what drove the observed trend.
First or last authorship changes, which we anticipated to affect gendered quotation rates, did not drive the downward trend.
Despite first authors more commonly being female than last authors, first and last authors were quoted at comparable rates over time.
Instead, column types were driving the trend.

> Our analysis shows that one article type, career-feature, has an equal number of quotes from both genders showing that gender parity is possible in science-related news coverage.

Reviewer: I think this is really a key point here, as is the bit above that notes that the introduction of this article type appears to be one of the major drivers in increased parity. Right now this paragraph starts with just a restating of results, so this key point gets buried - especially if someone reading this says: "eh, I've heard this before" and stops reading before the last few sentences. I would restructure each of these results-related paragraphs to put the primary point that goes beyond strict interpretation of the data first, and then provide the support within the paragraph.

Author: Yeah, I think this suggestion made the discussion sound much stronger. Edited in https://github.com/greenelab/nature_news_manuscript/pull/1/commits/6e11c15a9953873cfee4ddff90e8a5e7cf7a9ebb
